
<robot name="corner">
  <material name="Blue">
    <color rgba="0 0.5 1 1.0"/>
  </material>

  <!-- two connected boxes -->
  <link name="box1">
    <inertial>
      <mass value="10.0" />
      <inertia  ixx="1.0" ixy="0.0"  ixz="0.0"  iyy="1.0"  iyz="0.0"  izz="1.0" />
    </inertial>
    <visual>
      <geometry>
        <box size="5 0.1 2.5" />
      </geometry>
      <material name="Blue"/>
    </visual>
    <collision>
      <geometry>
        <box size="5 0.1 2.5" />
      </geometry>
    </collision>
  </link>
    <joint name="j_corner" type="fixed">
    <origin rpy="0 0 0" xyz="-2.5 -2.55 0.0"/>
    <parent link="box1"/>
    <child link="box2"/>
  </joint>
    <link name="box2">
    <inertial>
      <mass value="10.0" />
      <inertia  ixx="1.0" ixy="0.0"  ixz="0.0"  iyy="1.0"  iyz="0.0"  izz="1.0" />
    </inertial>
    <visual>
      <geometry>
        <box size="0.1 5.0 2.5" />
      </geometry>
      <material name="Blue"/>
    </visual>
    <collision>
      <geometry>
        <box size="0.1 5.0 2.5" />
      </geometry>
    </collision>
  </link>
</robot> 